apps/src/applab/Visualization.jsx

```diff
@@ -37,7 +48,7 @@ var Visualization = React.createClass({
     var appHeight = applabConstants.APP_HEIGHT - applabConstants.FOOTER_HEIGHT;
 
     return (
-      <ProtectedStatefulDiv>
+      <div>
```

Comment:
We could try to write more detailed protection, but maybe it's not necessary at this point.  We're getting _really_ close to not having external stuff mess with `#visualization` - the remaining cases I see in applab are the stylesheet-modification stuff, and design mode 'droppable'.

Do we need this outer `<div>` anymore?